scripts/test.py

```diff
@@ -204,6 +204,36 @@ def test(name, to_run, want, fail=False, sort=False):
   }
 }"""
 test("Prints schema as JSON", to_run, want, sort=True)
+
+# SQL file tests
+# Simple sql query from file
+to_run = """./dsq --file ./testdata/sql/simple.sql testdata/userdata.json"""
+want = """
+[{" Name ":"Michelle Yost"},
+{" Name ":"Guadalupe Schimmel II"},
+{" Name ":"Corey Beier"}]
+"""
+test("Run simple query from sql file", to_run, want, sort=True)
+
+# Join sql query from file
```

Comment:
I don't think these two tests after your first one test anything new right?

The only two other cases I can think of are testing the error message: `dsq x.json -f` (should return an error) and `dsq -f some_empty_file.sql` should return an error if the sql file is empty (I think this already happens naturally but we should just make sure)

Reply:
Yeah I'm just commenting about useful/not useful tests. This file is just tests